context-core/src/main/java/de/atb/context/persistence/common/RepositoryTDB.java

```diff
@@ -46,20 +52,20 @@ protected RepositoryTDB(final String baseLocation) {
 		super(baseLocation);
 	}
 
-	protected final synchronized boolean clearBaseDirectory() { // TODO this should maybe replace by a call to DRM API
+	protected final synchronized boolean clearBaseDirectory() {
 		for (final BusinessCase bc : BusinessCase.values()) {
 			clearBusinessCaseDirectory(bc);
 		}
 		return RepositoryTDB.clearDirectory(new File(basicLocation));
 	}
 
-	protected final synchronized boolean clearBusinessCaseDirectory( // TODO this should maybe replace by a call to DRM API
+	protected final synchronized boolean clearBusinessCaseDirectory(
 			final BusinessCase businessCase) {
 		return RepositoryTDB.clearDirectory(new File(
 				getLocationForBusinessCase(businessCase)));
 	}
 
-	protected static synchronized boolean clearDirectory(final File dir) { // TODO this should maybe replace by a call to DRM API
+	protected static synchronized boolean clearDirectory(final File dir) {
 		if (dir.isDirectory()) {
 			boolean oneFailed = false;
 			for (final String file : dir.list()) {
```

Comment:
[x] done

Reply:
`Objects.requireNonNull` throws NPE (= RuntimeException) if `dir.list` is `null`.

Will this exception be handled somewhere else so that it doesn't kill the whole app?

Reply:
ok, changed it